### PR TITLE
create layout home screen

### PR DESCRIPTION
**Pull request by Vanh1200**
## [Pull request title]
Create layout for home fragment
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7947
#### Screenshots (nếu có)
image

### CODE REVIEW

**Comment by Vanh1200** [comment]
ready

File: app/src/main/res/drawable/state_navigation_icon.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@color/color_dark_gray" android:state_checked="true" />
+    <item android:color="@android:color/black" />
+</selector>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/drawable/state_navigation_icon.xml]: need an endline

File: app/src/main/res/menu/menu_navigation.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        android:title="@string/tittle_home" />
+    <item
+        android:id="@+id/menu_favorite"
+        android:icon="@drawable/ic_heart"
+        android:title="@string/tittle_favorite" />
+    <item
+        android:id="@+id/menu_setting"
+        android:icon="@drawable/ic_setting"
+        android:title="@string/tittle_setting" />
+</menu>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/menu/menu_navigation.xml]: tương tự

File: app/src/main/res/values/dimens.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dp_1">1dp</dimen>
+    <dimen name="dp_200">200dp</dimen>
+    <dimen name="dp_10">10dp</dimen>
+    <dimen name="dp_5">5dp</dimen>
+    <dimen name="sp_26">26sp</dimen>
+</resources>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/values/dimens.xml]: tương tự

**Comment by Vanh1200** [comment]
ready

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,3 +1,9 @@
 <resources>
     <string name="app_name">Film Dictionary</string>
+    <string name="tittle_home">Home</string>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/values/strings.xml]: 3->6 sai tên rồi

**Comment by Vanh1200** [comment]
ready

**Pull request merged**